store/sqlstore/job_store.go

```diff
@@ -86,20 +88,21 @@ func (jss SqlJobStore) UpdateStatus(id string, status string) (*model.Job, *mode
 }
 
 func (jss SqlJobStore) UpdateStatusOptimistically(id string, currentStatus string, newStatus string) (bool, *model.AppError) {
-	var startAtClause string
+	sql := jss.getQueryBuilder().
+		Update("").
+		Table("Jobs").
+		SetMap(sq.Eq{"LastActivityAt": model.GetMillis(), "Status": newStatus}).
```

Comment:
Same here about using multiple times set instead of SetMap.

Reply:
Done.